.circleci/config.yml

```diff
@@ -85,12 +73,12 @@ workflows:
       - build377
       - build38
       - lint
-      - singularity-image:
+      - docker:
           requires:
             - build377
             - lint
           filters:
             branches:
               only:
                 - main
-                - /.*singularity.*/
+                - /.*dev.*/
```

Comment:
It looks like the docker build is ~ 6 minutes, down from ~9 for singularity. Should we just always build?
Or make a `no_docker` name flag?

Reply:
I still favor having user explicitly request for images to be built.

Reply:
sounds good